pyproject.toml

```diff
@@ -3,6 +3,28 @@ profile = "black"
 multi_line_output = 3
 order_by_type = false
 
+[tool.mypy]
```

Comment:
So we also have `.mypy.ini`

Why this change here?

Reply:
Sorry, I forgot to mention. While upgrading to mypy 0.91 and looking at docs I found that it now supports `pyproject.toml` so I thought I'd consolidate more tool config in there. I've actually deleted `.mypy.ini` in this PR. But I just realized I left out the comments from the previous config, I'll amend that.

Reply:
Ah. I somehow missed noticing the deletion of the file.